recipe/meta.yaml

```diff
@@ -29,6 +29,7 @@ requirements:
 test:
   imports:
     - imagej
+    - skip: True  # [osx]
```

Comment:
Doesn't this need to be one level up, in the `test:` section, not under `imports`?

Reply:
@imagejan -- good catch!